colcon_bundle/verb/_archive_generators.py

```diff
@@ -73,3 +61,88 @@ def generate_archive(install_base,
     os.remove(bundle_tar_path)
 
     logger.info('Archiving complete')
+
+
+def generate_archive_v2(install_base,
```

Comment:
Could we call it workspace_install_base for clarity?

Reply:
install_base is the terminology used by the rest of colcon and there is a flag `--install-base`. I am maintaining consistency with that upstream naming.